src/platform/forms-system/src/js/widgets/CurrencyWidget.jsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class CurrencyWidget extends React.Component {
+  constructor(props) {
+    super(props);
+    let value = props.value;
+    if (typeof value === 'number') {
+      value = value.toFixed(2);
+    }
+    this.state = {
+      value,
+    };
+  }
+
+  onBlur = () => {
+    this.props.onBlur(this.props.id);
+  };
+
+  handleChange = event => {
+    const val = event.target.value;
+    if (val === '' || typeof val === 'undefined') {
+      this.props.onChange();
+      // Needs to look like a currency
+    } else if (!/^\${0,1}[0-9,]*(\.\d{1,2})?$/.test(val)) {
```

Comment:
all of the linting issues were automatically fixed except this one. Can see the original here: https://github.com/usds/us-forms-system/blob/096f14f182500d3def179203d0c66c832a2d554d/src/js/widgets/CurrencyWidget.jsx#L27

eslint didn't like an orphan if inside of an else.